### PR TITLE
Django Tutorial Part 6: Generic list and detail views

### PR DESCRIPTION
**Pull request by tdanhjv**

### CODE REVIEW

**Comment by tdanhjv** [comment]
ready

**Comment by tdanhjv** [comment]
ready

File: locallibrary/catalog/admin.py [hunk 1]
```diff
@@ -38,4 +38,4 @@ class BookInstanceAdmin(admin.ModelAdmin):
         ('Availability', {
             'fields': ('status', 'due_back')
         }),
-    )
+    )
```
Comment (huongnt-2545) [inline locallibrary/catalog/admin.py]: thêm 1 dòng trống cuối file tránh warning từ github

File: locallibrary/catalog/templates/catalog/book_detail.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{% extends "base_generic.html" %}
+
+{% block content %}
+    <h1>Title: {{ book.title }}</h1>
```
Comment (huongnt-2545) [inline locallibrary/catalog/templates/catalog/book_detail.html]: translate text

File: locallibrary/catalog/templates/catalog/book_detail.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <p class="{% if copy.status == 'a' %}text-success{% elif copy.status == 'm'%}text-danger{% else %}text-warning{% endif %}">
+                {{ copy.get_status_display }}
+            </p>
+            {% if copy.status != 'a' %}
+                <p><strong>Due to be returned:</strong> {{ copy.due_back }}</p>
+            {% endif %}<p><strong>Imprint:</strong> {{ copy.imprint }}</p>
+            <p class="text-muted"><strong>Id:</strong> {{ copy.id }}</p>
+        {% endfor %}
+    </div>
+{% endblock %}
```
Comment (huongnt-2545) [inline locallibrary/catalog/templates/catalog/book_detail.html]: end of file -> thêm dòng trống cuối file

File: locallibrary/catalog/admin.py (hunk 1 above)
Comment (tdanhjv) [reply locallibrary/catalog/admin.py]: resolved

File: locallibrary/catalog/templates/catalog/book_detail.html (hunk 2 above)
Comment (tdanhjv) [reply locallibrary/catalog/templates/catalog/book_detail.html]: resolved

File: locallibrary/catalog/templates/catalog/book_detail.html (hunk 3 above)
Comment (tdanhjv) [reply locallibrary/catalog/templates/catalog/book_detail.html]: resolved

**Comment by tdanhjv** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
mớt

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**